src/components/ExerciseQuestion.tsx

```diff
@@ -129,39 +130,41 @@ export const ExerciseQuestion = React.forwardRef((props: ExerciseQuestionProps,
         displaySolution={displaySolution}
         show_all_feedback={show_all_feedback}
         tableFeedbackEnabled={tableFeedbackEnabled}
+        previewMode={previewMode}
       >
         <FreeResponseReview free_response={free_response} />
       </Question>
-      <StepCardFooter className="step-card-footer">
-        <div className="step-card-footer-inner">
-          <div className="points" role="status">
-            {available_points ? <strong>Points: {available_points}</strong> : null}
-            <span className="attempts-left">
-              {hasMultipleAttempts &&
-                attempts_remaining > 0 &&
-                <AttemptsRemaining count={attempts_remaining} />}
-            </span>
-            <PublishedComments published_comments={published_comments} />
-            {detailedSolution && (<div><strong>Detailed solution:</strong> <Content html={detailedSolution} /></div>)}
+      {(previewMode && detailedSolution) || !previewMode ?
```

Comment:
maybe I'm looking at the wrong place in figma, but I don't see a case where the detailed solution should be visible in the preview card. is there an example?

Reply:
we always want the feedback visible to instructors, the designs are wrong